Apps/APN/package.json

```diff
@@ -8,21 +8,24 @@
     "bundle:ios": "react-native bundle --entry-file index.js --platform ios --dev false --bundle-output ios/main.jsbundle --assets-dest ios",
     "clean-install": "./scripts/clean-install-dependencies.sh",
     "clean-install-android": "./scripts/clean-run-android.sh",
-    "clean-install-ios": "./scripts/clean-run-ios.sh"
+    "clean-install-ios": "./scripts/clean-run-ios.sh",
+    "dev:update": "../../scripts/dev:update.sh",
+    "postinstall": "pod update --project-directory=ios",
+    "preinstall": "npm run dev:update"
```

Comment:
I added these to make installing easier for devs. 

If someone was to clone the repo and try to run `npm install` for the first time for any of our sample apps, they would see errors because (1) `../../customerio-reactnative.tgz` does not yet exist and (2) `pod install` might have been forgetten to execute. 

By adding these scripts, by running `npm install`, we automatically solve both of those problems for you.